mhep/mhep/assessments/tests/views/test_retrieve_update_destroy_assessment.py

```diff
@@ -14,14 +14,9 @@ def setUpClass(cls):
         cls.me = UserFactory.create()
         super().setUpClass()
 
-    @classmethod
-    def tearDownClass(cls):
-        super().tearDownClass()
-        Assessment.objects.all().delete()
-
     def test_get_assessment(self):
         with freeze_time("2019-06-01T16:35:34Z"):
-            a = Assessment.objects.create(
+            a = AssessmentFactory.create(
                     owner=self.me,
                     name="test name",
```

Comment:
when using factories I think it's best to omit as much as possible e.g. `name`, `description`